vsphere-manager-blueprint.yaml

```diff
@@ -0,0 +1,858 @@
+      when `use_dhcp` is false
+    default: ''
+    type: string
+
+  management_network_ip:
+    description: >
+      server ip address. It will be used by the plugin only
+      when `use_dhcp` is false
+    default: ''
+    type: string
+
+  management_network_switch_distributed:
+    description: >
+       signifies if network is connected to a distributed switch
+    default: false
+    type: boolean
+
+  manager_server_domain:
+    description: >
+      the fully qualified domain name
+    default: ''
+    type: string
+
+  external_network_name:
+    description: >
+      network name
+    default: DMZ
+    type: string
+
+  external_network_use_dhcp:
+    description: >
+      use DHCP to obtain an ip address
+    default: true
+    type: boolean
+
+  external_network_cidr:
+    description: >
+      network cidr (for example, 10.0.0.0/24).
+      It will be used by the plugin only when `use_dhcp` is false
+    default: ''
+    type: string
+
+  external_network_gateway:
+    description: >
+      network gateway ip. It will be used by the plugin only when
+      `use_dhcp` is false
+    default: ''
+    type: string
+
+  external_network_ip:
+    description: >
+      server ip address. It will be used by the plugin only when
+      `use_dhcp` is false
+    default: ''
+    type: string
+
+  external_network_switch_distributed:
+    description: >
+      signifies if network is connected to a distributed switch
+    default: false
+    type: boolean
+
+  manager_server_cpus:
+    type: string
+    default: 2
+    description: >
+      Count cpu on node
+
+  manager_server_memory:
+    type: string
+    default: 4096
+    description: >
+      Manager host memory allocation
+
+  #############################
+  # vsphere Inputs
+  #############################
+
+  vsphere_username:
+    type: string
+    description: >
+      User login for vsphere
+
+  vsphere_password:
+    type: string
+    default: ''
+    description: >
+      User password for vsphere air
+
+  vsphere_host:
+    description: >
+      vSphere host
+    type: string
+
+  vsphere_port:
+    description: >
+      port which vCenter Server system uses to monitor data transfer
+      from SDK clients
+    default: 443
+    type: string
+
+  vsphere_datacenter_name:
+    description: >
+      datacenter name
+    default: Datacenter
+    type: string
+
+  vsphere_resource_pool_name:
+    description: >
+      Resource pool name. Represents a set of physical resources:
+      a single host, a subset of a host's resources, or resources
+      spanning multiple hosts
+    default: Resources
+    type: string
+
+  vsphere_auto_placement:
+    description: >
+      signifies if server is to be automatically placed on a host
+    default: false
+    type: boolean
+
+  template:
+    type: string
+    default: CentOS-7-x86_64-1406
+    description: >
+      Name of template,
+      can be 'CentOS-7.0-1406-x86_64-tmpl'
+
```

Comment:
all above inputs should be under "Provider specific inputs".. much like in any other manager blueprint.
Thank you.